src/app.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
leave the space at the end

Reply:
but there is a line on my end I don't know the issue